Composer/packages/lib/bot-deploy/src/botProjectDeploy.ts

```diff
@@ -4,7 +4,7 @@
 import * as path from 'path';
 import * as util from 'util';
 
-// import { WebSiteManagementClient } from '@azure/arm-appservice-profile-2019-03-01-hybrid';
+import { WebSiteManagementClient } from '@azure/arm-appservice-profile-2019-03-01-hybrid';
```

Comment:
Please back out this whole commit and go back to using the token.

As discussed in teams, use the error status on the zipdeploy -- if 403, this indicates a token failure.

Reply:
done